mcs/class/corlib/System.Reflection/RuntimeMethodInfo.cs

```diff
@@ -714,7 +714,8 @@ public override MethodInfo MakeGenericMethod (Type [] methodInstantiation)
 #if FULL_AOT_RUNTIME
 				throw new NotSupportedException ("User types are not supported under full aot");
```

Comment:
As commented else change this to
```c#
#if !FULL_AOT_RUNTIME
				if (RuntimeFeature.IsDynamicCodeSupported)
					return new MethodOnTypeBuilderInst (this, methodInstantiation);
#endif

				throw new NotSupportedException ("User types are not supported");      
```

Reply:
Oh sorry, I completely forgot this one.  It's now fixed.